src/shared.ts

```diff
@@ -0,0 +1,37 @@
+interface OptionsParam {
```

Comment:
Can we follow the same export pattern as in [src/helpers.ts](https://github.com/smileidentity/smile-identity-core-js/pull/58/files#diff-ca5696b367d474e785317cb7a0d9853fef5729387ab8d0fab4c46268c03aae99)